staging/src/k8s.io/component-base/logs/api/v1/options.go

```diff
@@ -93,8 +93,25 @@ func (c *LoggingConfiguration) Validate(featureGate featuregate.FeatureGate, fld
 			}
 		}
 	}
-	if _, err := logRegistry.get(c.Format); err != nil {
+	factory, err := logRegistry.get(c.Format)
+	if err != nil {
 		errs = append(errs, field.Invalid(fldPath.Child("format"), c.Format, "Unsupported log format"))
+	} else if factory != nil {
+		feature := factory.Feature()
+		if feature != "" {
+			enabled := featureGates()[feature].Default
+			if featureGate != nil {
+				enabled = featureGate.Enabled(feature)
+			}
+			if !enabled {
+				// Look up alpha/beta from actual feature. Should exist, but the API doesn't guarantee it.
+				status := "experimental"
+				if featureSpec, ok := featureGate.DeepCopy().GetAll()[feature]; ok {
+					status = string(featureSpec.PreRelease)
+				}
```

Comment:
is `feature` ever going to be anything other than `LoggingAlphaOptions` or `LoggingBetaOptions`? I'd recommend just saying "Log format %s is not available because feature %s is not enabled" instead of doing this lookup of the PreRelease field

Reply:
Agreed, adding the stability level doesn't add much additional information. Removed.